controls/datagrid/theming-and-styles/style-selectors.md

```diff
@@ -11,61 +11,107 @@ slug: datagrid-style-selectors
 * `CellDecorationStyleSelector`(`IStyleSelector`)&mdash;Styles the decoration of a cell.
 
-Different styles can be applied on a per-group header once the DataGrid control is grouped through `GroupHeaderStyleSelector` property.
+> For the DataGrid Style Selector example, go to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and navigate to the **DataGrid > Styling** category.
+
+## Row Background Style Selector
+
+You can set a different style on a row, alternate row, and on row details based on custom style-selection logic by using the `RowBackgroundStyleSelector` (`IStyleSelector`) property.
+
+> For the DataGrid Row Background Style Selector example, go to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and navigate to the **DataGrid > Styling** category.
 
-Different styles can be applied on a per-group footer once the DataGrid control is grouped through `GroupFooterStyleSelector` property.
+## Group Style Selector
+
+You can set a different style on a group header and footer based on custom style-selection logic with the following properties:
+
+* `GroupHeaderStyleSelector`(`IStyleSelector`)&mdash;Specifies different style per-group header once the DataGrid control is grouped.
+* `GroupFooterStyleSelector`(`IStyleSelector`)&mdash;Specifies different style per-group footer once the DataGrid control is grouped.
 
 > To display the group footer, set the `ShowGroupFooters` property to `True`.
 
-The `CellContentStyleSelector`, `CellDecorationStyleSelector`, and `GroupStyleSelector` use the `SelectStyle` method to change the style.
+> For the DataGrid Style Selector example, go to the [SDKBrowser Demo Application]({%slug sdkbrowser-app%}) and navigate to the **DataGrid > Styling** category.
 
-## Example
+The `CellContentStyleSelector`, `CellDecorationStyleSelector`, `RowBackgroundStyleSelector`, `GroupHeaderStyleSelectorand`, and `GroupFooterStyleSelector` use the `SelectStyle` method to change the style.
 
-The following example will demonstrate how to apply the style selectors in the DataGrid control:
+## Example with Cell and Group Style Selectors
 
-Let’s add the DataGrid and set the `CellContentStyleSelector` as a static resource of type `MyCellContentStyleSelector`, `CellDecorationStyleSelector` as a static resource of type `MyCellDecorationStyleSelector`, and `GroupStyleSelector` as a static resource of type `MyGroupStyleSelector`.
+The following example will demonstrate how to apply the style selectors on the DataGrid cell and group header:
 
-**1.** `DataGrid` definition in XAML:
+Let’s add the DataGrid and set the `CellContentStyleSelector` property as a static resource of type `MyCellContentStyleSelector`, `CellDecorationStyleSelector` as a static resource of type `MyCellDecorationStyleSelector`, and `GroupStyleSelector` as a static resource of type `MyGroupStyleSelector`.
```

Comment:
This describes in more detail what the example shows, so it belongs to the paragraph above (it serves as an introduction for the example).
```suggestion
```